packages/rich-text/src/test/rich-text.perf-native.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import {
+	changeTextOfRichText,
+	fireEvent,
+	measurePerformance,
+	screen,
+} from 'test/helpers';
+
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+
+describe( 'RichText Performance', () => {
+	it( 'performance is stable when typing using Rich Text', async () => {
```

Comment:
Suggestion to balance clarity and brevity. Feel free to ignore.
```suggestion
describe( 'RichText', () => {
	it( 'performance is stable when typing text', async () => {
```